lib/internal/util.js

```diff
@@ -262,10 +262,13 @@ function slowCases(enc) {
   }
 }
 
-function emitExperimentalWarning(feature) {
+function emitExperimentalWarning(feature, messageBefore) {
```

Comment:
I think this naming is better
```suggestion
function emitExperimentalWarning(feature, messagePrefix) {
```

Reply:
Or `warningHint`?